shopinvader_api_cart/routers/cart.py

```diff
@@ -43,17 +44,30 @@ def sync(
     env: Annotated[api.Environment, Depends(authenticated_partner_env)],
     partner: Annotated["ResPartner", Depends(authenticated_partner)],
     uuid: str | None = None,
-) -> CartResponse | None:
+) -> Sale | None:
```

Comment:
@sebastienbeau a `Cart` is not a `Sale`. Even if both models share common data, we will end up  with specific data depending of the context where it's use. Someone will want to add a field used when the sale is a quotation, someone else an other one when the sale is confirmed etc....  Can you describe why you want only one model for SO, Quotation, Cart etc....

Reply:
@thibaultrey would like to have the same model for representing a Sale Order, a Quotation, a Cart as at the end is the same thing, for sure some field can be "empty" when it's a cart. But this may simplify the life for frontend developer.

@thibaultrey can you share more feedback ?

Reply:
You can have the same common schema. But I would like to be able to define specific constrains in the context of a cart or so or quotation without breaking others usages. For example, the billing information could be a required field on a Sale Order but can't be required on a Cart.